.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -43,6 +43,17 @@ body:
       - arm64 (Apple Silicon)
   validations:
     required: true
+- type: dropdown
+  attributes:
+    label: Linux only: what package format did you use to install Rancher Desktop?
```

Comment:
There's an error pops up on this line: https://github.com/rancher-sandbox/rancher-desktop/blob/faa39cfc3b8ca863b75d830a65a124f8e58c5999/.github/ISSUE_TEMPLATE/bug_report.yml#L48

`YAML error: (): mapping values are not allowed in this context at line 48 column 22.`